src/components/Card-Product/v-PaginateModul.vue

```diff
@@ -31,7 +31,7 @@
     },
     data(){
       return {
-        numb: this.modelValue as number,
+        number_page: this.modelValue as number,
```

Comment:
number_page: null as Nullable<number>,
methods - initialState() {
this.number_page = this.modelValue
}

mounted - this.initialState